### PR TITLE
PROPOSE: RFC0511 register DIF credential-manifest as attachment format in issue-credential V2

### PR DESCRIPTION
**Pull request by llorllale**
Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: features/0511-dif-cred-manifest-attach/README.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+{
+    "challenge": "...",
+    "domain": "...",
+    "credential_manifest": {
+        // credential manifest object
+    }
+}
+```
+
+Complete message example:
```
Comment (swcurran) [inline features/0511-dif-cred-manifest-attach/README.md:93]: I'm thinking a better approach would be to have a pointer to examples in the DIF document vs. putting examples inline here.  I think this document should provide context between the Issue Credential protocol and the DIF data model - how one is used with the other.

File: features/0511-dif-cred-manifest-attach/README.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+                }
+            }
+        }
+    }]
+}
+```
+
+### `issue-credential` attachment format
+
+This specification does not register any format identifier for the `issue-credential` message. The Issuer SHOULD set the `format` to the value
```
Comment (swcurran) [inline features/0511-dif-cred-manifest-attach/README.md:261]: I don't know that much about the DIF document, but I don't understand why there would not be a format string for this.

File: features/0511-dif-cred-manifest-attach/README.md (hunk 1 above)
Comment (TelegramSam) [reply features/0511-dif-cred-manifest-attach/README.md:93]: I think an intentionally incomplete example is useful. Show some outer structure, and indicate the incompleteness with a comment.

**Comment by swcurran** [comment]
Hmm...no button to update the base branch. Back to you @llorllale

**Comment by llorllale** [comment]
@swcurran updated - please check now

**Pull request merged**